tests/test_factory_build.py

```diff
@@ -85,3 +85,17 @@ class MyFactory(ModelFactory):
 
     with pytest.raises(ConfigurationError):
         MyFactory.build(factory_use_construct=True)
+
+
+def test_build_instance_with_allow_population_by_field_name_flag():
```

Comment:
We should test both cases with the alias and the field_name, showing that the `build` method accepts both keys.